tztrout/data.py

```diff
@@ -90,8 +97,27 @@ def deduplicate(item):
     return deduplicate
 
 
+def generate_us_zipcode_namedtuples():
+    """Generate a list of namedtuples for US zipcodes from the list of
+    dictionaries in data/us_zipcode_data.json
+    """
+    if not os.path.exists(US_ZIPCODE_DATA_PATH):
+        raise ValueError(
+            'US Zipcode data missing. Run regenerate_data.py first'
+        )
+    us_zipcode_tuples = []
+    with open(US_ZIPCODE_DATA_PATH, 'r') as file:
+        data = json.loads(file.read())
```

Comment:
`json.read(file)` to avoid loading the whole thing in memory as a string.

Reply:
I assume you mean `json.load` here.` json.read` does not exist. Done in 79b5bd4

Reply:
Yep, you're right.